apps/scripts/AliveTest.c

```diff
@@ -0,0 +1,143 @@
+// 4KB of D_MEM
+// 2048 - 4096 - Shared
+//
+// 1536 -    2048 - Thread 1
+// 1024 -    1536 - Thread 1
+// 512  -    1024 - Thread 1
+// 0    -    512  - Thread 0
+
+// REGION == 2'b01;
+
+#define D_MEM_SHARED (*(volatile int (*)[64])(0x00401000))
```

Comment:
You can allocate the 2KB of Shared Memory.
64 int (256 Byte) you allocated is just the beginning of the Shared memory

Reply:
This is very strange syntax. Something like this should be enough:
```c
#define D_MEM_SHARED ((volatile int *) (0x00401000))
```